sdk/communication/AzureCommunicationCommon/AzureCommunicationCommon.podspec.json

```diff
@@ -24,6 +24,10 @@
   "pod_target_xcconfig": {
     "DEFINES_MODULE": "YES"
   },
-  "dependencies": {},
+  "dependencies": {
+      "AzureCore": [
+        "1.0.0-beta.15"
+      ]
+  },
```

Comment:
Common should not take a dependency on AzureCore because it is used by Calling (ObjC based) and AzureCore is fundamentally not compatible with ObjC codebases.

Reply:
Thanks, fixed